public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -141,6 +145,76 @@ export function transformSceneToSaveModelSchemaV2(scene: DashboardScene, isSnaps
   }
 }
 
+function getLayout(
+  layoutManager: DashboardLayoutManager,
+  isSnapshot?: boolean
+): GridLayoutKind | RowsLayoutKind | ResponsiveGridLayoutKind {
+  if (layoutManager instanceof DefaultGridLayoutManager) {
```

Comment:
Every layout can implement a `toSaveModel()` method according to `public/app/features/dashboard-scene/scene/types.ts`. This is a good opportunity to also remove the optionality of that method.

Reply:
Yes I think this makes sense. I suggest we add this as a follow up ticket on this and we can start refactoring.